generated/nixnetsocket/nixnetsocket.proto

```diff
@@ -42,11 +42,45 @@ enum NiXnetSocketAttribute {
   NIXNETSOCKET_ATTRIBUTE_UNSPECIFIED = 0;
 }
 
+enum AddressFamily {
+  ADDRESS_FAMILY_UNSPECIFIED = 0;
+  ADDRESS_FAMILY_INET = 2;
+  ADDRESS_FAMILY_INET6 = 10;
+}
+
+enum IPProtocol {
+  IP_PROTOCOL_IP = 0;
+  IP_PROTOCOL_TCP = 6;
+  IP_PROTOCOL_UDP = 8;
+  IP_PROTOCOL_IPV6 = 12;
+}
+
+enum SocketProtocolType {
+  SOCKET_PROTOCOL_TYPE_UNSPECIFIED = 0;
+  SOCKET_PROTOCOL_TYPE_STREAM = 1;
+  SOCKET_PROTOCOL_TYPE_DGRAM = 2;
+}
+
+enum SocketOptionLevel {
+  SOCKET_OPTION_LEVEL_UNSPECIFIED = 0;
+  SOCKET_OPTION_LEVEL_SOCKET = 13;
+}
+
 enum OptName {
+  option allow_alias = true;
   OPT_NAME_UNSPECIFIED = 0;
   OPT_NAME_TCP_NODELAY = 1;
-  OPT_NAME_IP_MULTICAST_TTL  = 6;
+  OPT_NAME_IP_ADD_MEMBERSHIP = 3;
```

Comment:
Note: We'll need to handle these new OptName values in the conversion code but I don't think that should be part of this change. Added AB#1889111 to address that.